app/utils/metrics.js

```diff
@@ -20,13 +20,19 @@ import _ from 'lodash';
 const NONE_PROVIDED = 'No Event Name Provided';
 
 export function createMetricsTracker(api) {
-  return ({ getState }) => (next) => (action) => {
+  return () => (next) => (action) => {
     if (_.get(action, 'meta.metric', null) !== null) {
       api.metrics.track(
         _.get(action, 'meta.metric.eventName', NONE_PROVIDED),
         _.get(action, 'meta.metric.properties', {})
       );
     }
+		if (_.get(action, 'payload.state.meta.metric', null) !== null) {
```

Comment:
This is to extract meta information from the route change/page navigation events.